static/app/components/workflowEngine/form/inlineNumberField.tsx

```diff
@@ -0,0 +1,18 @@
+import type {ComponentProps} from 'react';
+import styled from '@emotion/styled';
+
+import NumberField from 'sentry/components/forms/fields/numberField';
```

Comment:
🤔 cc @natemoo-re any reason that we have these fields as separate vs just using the input type?

Reply:
Yep! This mirrors our form primitives and the `NumberField` supports a few extra properties for entering integer values like an inline unit `suffix` and improved `step` behavior.